code/modules/mod/mod_activation.dm

```diff
@@ -0,0 +1,149 @@
+		to_chat(wearer, "<span class='notice'>The [boots.name] [active ? "relax their grip on your legs" : "seal around your feet"].</span>")
+		boots.icon_state = "[skin]-boots[active ? "" : "-sealed"]"
+		boots.worn_icon_state = "[skin]-boots[active ? "" : "-sealed"]"
+		wearer.update_inv_shoes()
+		playsound(src, 'sound/mecha/mechmove03.ogg', 25, TRUE)
+	if(do_after(wearer,2 SECONDS,wearer,IGNORE_USER_LOC_CHANGE|IGNORE_TARGET_LOC_CHANGE|IGNORE_HELD_ITEM|IGNORE_INCAPACITATED))
+		to_chat(wearer, "<span class='notice'>The [gauntlets.name] [active ? "become loose around your fingers" : "tighten around your fingers and wrists"].</span>")
+		gauntlets.icon_state = "[skin]-gauntlets[active ? "" : "-sealed"]"
+		gauntlets.worn_icon_state = "[skin]-gauntlets[active ? "" : "-sealed"]"
+		wearer.update_inv_gloves()
+		playsound(src, 'sound/mecha/mechmove03.ogg', 25, TRUE)
+	if(do_after(wearer,2 SECONDS,wearer,IGNORE_USER_LOC_CHANGE|IGNORE_TARGET_LOC_CHANGE|IGNORE_HELD_ITEM|IGNORE_INCAPACITATED))
+		to_chat(wearer, "<span class='notice'>The [chestplate.name] [active ? "releases your chest" : "cinches tight again your chest"].</span>")
+		chestplate.icon_state = "[skin]-chestplate[active ? "" : "-sealed"]"
+		chestplate.worn_icon_state = "[skin]-chestplate[active ? "" : "-sealed"]"
+		if(active)
+			chestplate.clothing_flags &= ~chestplate.visor_flags
+			chestplate.flags_inv &= ~chestplate.visor_flags_inv
+		else
+			chestplate.clothing_flags |= chestplate.visor_flags
+			chestplate.flags_inv |= chestplate.visor_flags_inv
+		wearer.update_inv_wear_suit()
+		wearer.update_inv_w_uniform()
+		playsound(src, 'sound/mecha/mechmove03.ogg', 25, TRUE)
+	if(do_after(wearer,2 SECONDS,wearer,IGNORE_USER_LOC_CHANGE|IGNORE_TARGET_LOC_CHANGE|IGNORE_HELD_ITEM|IGNORE_INCAPACITATED))
+		to_chat(wearer, "<span class='notice'>The [helmet.name] hisses [active ? "open" : "closed"].</span>")
+		helmet.icon_state = "[skin]-helmet[active ? "" : "-sealed"]"
+		helmet.worn_icon_state = "[skin]-helmet[active ? "" : "-sealed"]"
+		if(active)
+			helmet.flags_cover &= ~helmet.visor_flags_cover
+			helmet.flags_inv &= ~helmet.visor_flags_inv
+			helmet.clothing_flags &= ~helmet.visor_flags
+			helmet.alternate_worn_layer = initial(helmet.alternate_worn_layer)
+		else
+			helmet.flags_cover |= helmet.visor_flags_cover
+			helmet.flags_inv |= helmet.visor_flags_inv
+			helmet.clothing_flags |= helmet.visor_flags
+			helmet.alternate_worn_layer = null
+		wearer.update_inv_head()
+		wearer.update_inv_wear_mask()
+		wearer.update_hair()
+		playsound(src, 'sound/mecha/mechmove03.ogg', 25, TRUE)
+	if(do_after(wearer,2 SECONDS,wearer,IGNORE_USER_LOC_CHANGE|IGNORE_TARGET_LOC_CHANGE|IGNORE_HELD_ITEM|IGNORE_INCAPACITATED))
+		audible_message("<span class='notice'>Systems [active ? "shut down. Parts unsealed. Goodbye" : "started up. Parts sealed. Welcome"], [wearer.real_name].</span>", hearing_distance = 1)
```

Comment:
can this reveal your true identity to people who are standing next to you or on your tile?

Reply:
maybe